lib/circuitbox/store.rb

```diff
@@ -0,0 +1,83 @@
+require "delegate"
+
+class Circuitbox
```

Comment:
I'd rather see a moneta middleware than a wrapper.

With middleware, it is easy to see in one place all the middleware used and understand the behaviour.
With factory wrapping like here, when you have more than one, understanding final behaviour become difficult.